README.md

```diff
@@ -1 +1,52 @@
-# Sweetsour Parser
+<p align="center">
+  <img alt="Sweetsour" src="/docs/logo.png?raw=true" width="200px" />
+</p>
+<h1 align="center">Sweetsour</h1>
+<p align="center">
+  A CSS-in-JS parser and pipeline adhering to the ISTF spec 🍭
+</p>
+
+<br />
+
+This is a reference implementation of a CSS-in-JS parser, outputting ISTF, according
+to the current [ISTF specification](https://github.com/cssinjs/istf-spec).
+
+## Status
+
+**Work in Progress: Not ready for any use!**
+
+Currently work is ongoing to build: **The parser (~60%)**
+
+- [ ] Parser
+- [ ] Transform tool with plugin support
+- [ ] Reference Babel plugin
+- [ ] Flattener for nested CSS (plugin)
+- [ ] Static prefixer (plugin)
```

Comment:
Should we completely separate parser and preprocessing pipelines?

Reply:
I could imagine a plugins system which takes ISTF and outputs ISTF and can be used as a pre or post processor or even at runtime, it doesn't matter, you just pick plugins.

Reply:
Yea, the reason why the transform tool is separate to be completely independent from the parser; I’ll have to check though how feasible this is